appendix-basic65.tex

```diff
@@ -9367,8 +9485,10 @@ \subsection{SPC}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$A6
 \item [Format:] {\bf SPC(}columns{\bf)}
-\item [Usage:] Skips {\bf columns}. \\
-               The effect is similar to pressing \megakey{$\rightarrow$}
+\item [Usage:] A string of cursor-right PETSCII codes, suitable for printing
+               to advance the cursor the given number of columns.
```

Comment:
Describing it this way made me curious to try if I could do the following: `A$=SPC(10)`, though that gave me a syntax error.

So it seems SPC is exclusively used by PRINT? If so, I'm wondering if this line should explicitly mention it.

Reply:
Actually an error in this entry! It's SPC$(), not SPC(). 😅 Fixed.

Reply:
Oh wait, I gave `PRINT SPC$(10);"HELLO"` a try and it didn't produce any spaces though.

I took a look back at the C64 reference manual, and it was listed there as SPC() also (not SPC$), so it genuinely does seem to be SPC()! (though I agree this does seem to break a convention of string-related functions ending with $...

So feel like in your more recent commit, you'll need to revert from SPC$ back to SPC.

And I suppose this SPC() just is so unique, that doing an `A$=SPC(10)` is not permissible for some reason, and it only works with `PRINT` :(

Reply:
sorry I confused myself because A$=SPC$(10) didn't produce an error, but that's because it's accessing undimensioned array SP$ which has 10 elements. I'll fix it back. Seemed like a weird thing to be wrong. :)